docroot/themes/custom/uids_base/templates/navigation/top-scroll.html.twig

```diff
@@ -0,0 +1,10 @@
+{{ attach_library('uids_base/top-scroll') }}
+  <nav class="top-scroll-container js-scroll" id="top-scroll" role="navigation" aria-label="Navigation">
```

Comment:
Can the aria-label be changed to something more representative of the action? "Back to top of page"

Reply:
@joewhitsitt GOOD IDEA! Sure thing! P.S. Adding a focus/hover state to button and still needing to probably remove code from uiowa_core.module that had code needed for page by page stuff.